lib/node/nodes/aggregate-intersection.js

```diff
@@ -41,10 +51,20 @@ function setAliasPrefixToColumnNames(columnNames, alias) {
     }).join(', ');
 }
 
+var densityQueryTemplate = Node.template([
+    ',',
+    '{{=it.aggregateFunction}}({{=it.qualifiedAggregateColumn}})',
+    '/',
+    // use 2.6e-06 as minimum area (tile size at zoom level 31)
+    'GREATEST(0.0000026, ST_Area((ST_Transform(_cdb_analysis_source.the_geom, 4326))::geography))',
```

Comment:
@javisantana told me, because `2.6e-06` is the minimum area (zoom 31) for the pixel size (if I understood correctly)